src/dropin.js

```diff
@@ -19,6 +19,86 @@ var svgHTML = fs.readFileSync(__dirname + '/html/svgs.html', 'utf8');
 var DEFAULT_CHECKOUTJS_LOG_LEVEL = 'warn';
 var VERSION = process.env.npm_package_version;
 
+/**
+ * @typedef {object} Dropin~cardPaymentMethodPayload
+ * @property {string} nonce The payment method nonce.
+ * @property {object} details Additional account details.
+ * @property {string} details.cardType Type of card, ex: Visa, MasterCard.
+ * @property {string} details.lastTwo Last two digits of card number.
+ * @property {string} description A human-readable description.
+ * @property {string} type The payment method type, always `CreditCard` when the method requested is a card.
+ */
+
+/**
+ * @typedef {object} Dropin~paypalPaymentMethodPayload
+ * @property {string} nonce The payment method nonce.
+ * @property {object} details Additional PayPal account details. See a full list of details in the [PayPal client reference](http://braintree.github.io/braintree-web/{@pkg bt-web-version}/PayPalCheckout.html#~tokenizePayload).
+ * @property {string} type The payment method type, always `PayPalAccount` when the method requested is a PayPal account.
+ */
+
+/**
+ * @name Dropin#on
+ * @function
+ * @param {string} event The name of the event to which you are subscribing.
```

Comment:
I just did this the same way as we did in Hosted Fields, down to update it in both places though.